extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ObjectType.scala

```diff
@@ -33,6 +33,7 @@ object ObjectType extends Enumeration {
       case PrivilegeObjectType.TABLE_OR_VIEW if opType.toString.contains("VIEW") => VIEW
       case PrivilegeObjectType.TABLE_OR_VIEW => TABLE
       case PrivilegeObjectType.FUNCTION => FUNCTION
+      case PrivilegeObjectType.DFS_URL | PrivilegeObjectType.LOCAL_URI => URI
```

Comment:
I don't know why we separate DFS_URL from LOCAL_URI but then merge them as the same again

Reply:
Just follow the hive's code